sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
+# input is a board
```

Comment:
Need a validate_sudoku_test.py file that tests the validate function with different inputs.